app/helpers/users_helper.rb

```diff
@@ -11,4 +11,9 @@ def reinvite_user_link(user)
       link_to tag.i("", class: 'fa fa-envelope', alt: "Re-send invitation", title: "Re-send invitation"), resend_user_invitation_organization_path(user_id: user.id), method: :post
     end
   end
+
+  def get_time_at_timezone
```

Comment:
Is this really part of the User routes? Or is this specific to the Devise mailer and should be in the Devise helper?